content/using/install.md

```diff
@@ -127,9 +127,26 @@ Either command will create a directory called `sampel-palnet/` and begin buildin
 
 When your ship is finished booting, you will see either the `~sampel-palnet:dojo>` or `~sampel-palnet:chat-cli/` prompt. If you're seeing `:chat-cli` press `Ctrl-x` to switch into Dojo. At that point, you should permanently erase your keyfile from your machine.
 
-To shut down your ship, use `Crtl-d`. 
+To shut down your ship, use `Ctrl-d`. 
 
-To start your ship up again, run `./urbit sampel-palnet` from the directory where your Urbit binary is saved. Don't boot multiple instances of your ship at the same time.
+To start your ship up again, run `./urbit sampel-palnet/` from the directory
+where your Urbit binary is saved. Note that in this usage,
+`sampel-palnet/` is the path of a folder, which by default is located in the same folder as
+the Urbit binary. This folder is called your ship's **pier**, and you
+can put it wherever you like.
+
+Never boot multiple instances of your ship at the same time. You can prevent
+this from happening on accident by only ever keeping a single copy of your pier.
+
+### Updating to the latest binary
+
+Most updates to Urbit are downloaded and applied automatically as OTA (Over the
+Air) updates. Occasionally it would be infeasible to distribute an update this
+way, and a new `urbit` binary is released. This is announced in the `urbit-dev`
```

Comment:
you should probably link the urbit-dev group when you mention it : https://groups.google.com/a/urbit.org/forum/#!forum/dev